django_afip/models.py

```diff
@@ -1708,3 +2017,55 @@ def __repr__(self) -> str:
     class Meta:
         verbose_name = _("receipt validation")
         verbose_name_plural = _("receipt validations")
+
+
+class CaeaCounter(models.Model):
+
+    pos = models.ForeignKey(
+        PointOfSales, related_name="counter", on_delete=models.PROTECT
+    )
+
+    receipt_type = models.ForeignKey(
+        ReceiptType, related_name="counter", on_delete=models.PROTECT
+    )
+
+    next_value = models.BigIntegerField(default=1)
+
+    class Meta:
+        constraints = [
+            models.UniqueConstraint(
+                fields=["pos", "receipt_type"],
+                name="unique_migration_pos_receipt_combination",
+            )
+        ]
+
+    def __str__(self):
+        return "Counter for POS:{}, receipt_type:{}. Next_value is {}".format(
+            self.pos, self.receipt_type, self.next_value
+        )
+
+
+class InformedCaeas(models.Model):
+
+    pos = models.ForeignKey(
+        PointOfSales, related_name="informed", on_delete=models.PROTECT
+    )
+
+    caea = models.ForeignKey(Caea, related_name="informed", on_delete=models.PROTECT)
+
+    processed_date = models.DateField(
+        _("processed date"),
+    )
+
+    def __str__(self):
+        return "POS:{}, with CAEA:{}, informed as without movement in {}".format(
```

Comment:
String is wrong here, there might have been movements.

Reply:
But you only inform CAEAs without movement, if a CAEA has movement it should be informed and in that case AFIP reject that operation. So you have `InformedCaeas` only with CAEAs without movement.

Reply:
How do we confirm if a `Caea` _with_ movements has been informed to AFIP?

Reply:
AFIP not provide any method for that, it's directly related to the action of validate a receipt. Once you validated a receipt it seems that AFIP mark the Caea as used and if you tried to informed as without movement it will return an error.